examples/axis-color/index.js

```diff
@@ -0,0 +1,28 @@
+const container = document.querySelector('d3fc-svg');
+const margin = 10;
+
+const scale = d3
+    .scaleLinear()
+    .domain([0, 140])
+    .nice();
+
+const axis = fc.axisBottom(scale).decorate(s => {
+    s.enter()
+        .select('text')
+        .style('fill', d => (d >= 100 ? 'red' : 'black'));
+});
+
+d3.select(container)
+    .on('draw', () => {
+        d3.select(container)
+            .select('svg')
+            .append('g')
+            .attr('transform', 'translate(0, 10)')
```

Comment:
I'm not sure what this transform is for? I can understand if we're positioning a few on screen or shifting stuff around to be visible.